shared/src/commonMain/kotlin/com/kcc/kmmhackathon/shared/utility/extensions/Int+Extensions.kt

```diff
@@ -0,0 +1,7 @@
+package com.kcc.kmmhackathon.shared.utility.extensions
+
+fun Int.getPortionsString(): String = when (this) {
+    in 2 .. Int.MAX_VALUE -> "$this portions remaining"
+    1 -> "$this portion remaining"
+    else -> "Reserved"
```

Comment:
I think I preferred this when it was:
```
0 -> 
1 ->
else ->
```

If we prevent uploading a meal with a negative value. And we check before we PATCH, (and I _think_ the app would crash if we actually tried to PATCH to a negative number anyway) then I think that is safe enough

Reply:
👍 Fair enough, it is a kind of impossible state for the quantity to be in.